src/MSALWrapper/ExceptionListToStringConverter.cs

```diff
@@ -74,11 +74,19 @@ private static string SingleLineException(Exception ex)
         {
             if (ex != null)
             {
-                var message = ex.Message
-                                .Replace("\n", string.Empty)
-                                .Replace("\r", string.Empty)
-                                .Replace("\t", string.Empty);
-                return $"{ex.GetType()}: {message}";
+                IEnumerable<Exception> innerExceptions = ExceptionExtensions.GetAllExceptions(ex);
+                var messages = new List<string>();
+
+                foreach (Exception innerException in innerExceptions)
+                {
+                    var message = innerException.Message
+                                    .Replace("\n", string.Empty)
+                                    .Replace("\r", string.Empty)
+                                    .Replace("\t", string.Empty);

```

Comment:
I don't know that I ever bothered to look before now. Is there a reason we get rid of all these characters? Why do we make a single line exception? 🤔

Reply:
We wanted to single line exception to have just one value in the telemetry for all the exception messages. Otherwise, each exception message needs to be associated with a unique key in the telemetry.

I guess getting rid of the tabs and lines is to have trimmed exception messages and make them easy to read in telemetry?

Reply:
Yes, this is about making it clear how many exceptions are getting logged. We could alternatively format them as a JSON array or something but that would take longer.